macos/FluentUITestViewControllers/TestButtonViewController.swift

```diff
@@ -159,9 +159,15 @@ class TestButtonViewController: NSViewController, NSMenuDelegate {
 		let leadingArrowImage = NSImage(named: TestButtonViewController.leadingArrow)!
 		let trailingArrowImage = NSImage(named: TestButtonViewController.trailingArrow)!
 
+		let customButton = Button()
+		customButton.title = "Custom Button"
+		customButton.image = NSImage(named: TestButtonViewController.nonTemplateImage)!
+		customButton.accentColor = communicationBlue
+		customButton.size = .large
+		customButton.style = .primary
 		let buttonsWithTitleAndImageHorizontal: () -> [Button] = {
 			return [
-				Button(title: "Back", image: leadingArrowImage, imagePosition: .imageLeading, format: largePrimary),
+				customButton,
```

Comment:
Should we have a customButton for all configurations (image only, title only, title and image)?

Reply:
The unique thing about this button is the non-template image (and the way it's constructed using the empty `Button()` init).  The other configurations are already covered in the test app.

Reply:
i.e. instead of modifying one button in the top row, make a new row where all the buttons are constructed in this way

Reply:
I have some updates ready for the test app that will incorporate this suggestion.  Feel free to leave it as-is here and I will submit a follow-up PR.

Reply:
So I should revert my changes in the test controller then?

Reply:
You don't need to.  It's fine to leave them as they are.